todo/src/AddToDo.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { createTODO } from "./actions";
+
+class AddAnimals extends Component {
```

Comment:
If you want to refer to other project code while you're working, that's fine... but remember to look out for these names, and please refrain from copying and pasting directly.